operator/pkg/central/extensions/reconcile_scanner_v4_db_password.go

```diff
@@ -0,0 +1,29 @@
+package extensions
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"github.com/operator-framework/helm-operator-plugins/pkg/extensions"
+	platform "github.com/stackrox/rox/operator/apis/platform/v1alpha1"
+	commonExtensions "github.com/stackrox/rox/operator/pkg/common/extensions"
+	ctrlClient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	_ commonExtensions.ScannerV4BearingCustomResource = (*platform.Central)(nil)
```

Comment:
I think this is correct at this position since this assignment is meant to make sure central is an implementation of `commonExtensions.ScannerV4BearingCustomResource`

Reply:
Going to close this to merge, since this is a minor detail and you already approved.